datarobot_provider/example_dags/datarobot_custom_job_dag.py

```diff
@@ -0,0 +1,92 @@
+# Copyright 2024 DataRobot, Inc. and its affiliates.
+#
+# All rights reserved.
+#
+# This is proprietary source code of DataRobot, Inc. and its affiliates.
+#
+# Released under the terms of DataRobot Tool and Utility Agreement.
+
+from datetime import datetime
+
+from airflow.decorators import dag
+
+from datarobot_provider.operators.custom_job import CreateCustomJobOperator
+from datarobot_provider.operators.custom_job import AddFilesToCustomJobOperator
+from datarobot_provider.operators.custom_job import SetCustomJobExecutionEnvironmentOperator
+from datarobot_provider.operators.custom_job import SetCustomJobRuntimeParametersOperator
+from datarobot_provider.operators.custom_job import RunCustomJobOperator
+from datarobot_provider.sensors.client import BaseAsyncResolutionSensor
+
+
+@dag(
+    schedule=None,
+    start_date=datetime(2023, 1, 1),
+    tags=['example', 'custom job'],
+    params={},
+)
+def create_custom_custom_job():
+    create_custom_job_op = CreateCustomJobOperator(
```

Comment:
Does it make sense to create another kind of create operator which includes files, environment and runtime parameters all in one? This way the DAG could be simplified. Having these more specific operators makes sense, but I think a super-creator operator could be useful.